src/features/toilet/toiletSlice.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+import { createSlice } from "@reduxjs/toolkit";
+
+export const toiletSlice = createSlice({
+  name: "currnetToiletInfo",
+  initialState: {
+    allIds: [],
+    byIds: {},
+  },
+  reducers: {
+    updateToiletInfo: (state, action) => {
```

Comment:
액션의 이름은 저는 event driven 하게 작성하는 편입니다. 여기서는 toiletInfoUpated 정도가 되겠네요! 이유는 아래의 리덕스 공식 문서를 참조 해주세요!

```suggestion
    toiletInfoUpated: (state, action) => {
```

https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters

Reply:
해당사항 수정하겠습니다!